### PR TITLE
feat(billing): implement wallet type switch

### PR DESCRIPTION
**Pull request by ygrishajev**
refs #247

### CODE REVIEW

File: apps/deploy-web/src/context/CertificateProvider/CertificateProviderContext.tsx [hunk 1]
```diff
@@ -152,22 +151,20 @@ export const CertificateProvider = ({ children }) => {
...
-        setLocalCert(_cert as LocalCert);
+      if (wallet.address === address) {
+        setLocalCert(cert);
       }
-    }
+
+      return acc;
+    }, [] as LocalCert[]);
 
     setLocalCerts(certs);
```
Comment (baktun14) [inline apps/deploy-web/src/context/CertificateProvider/CertificateProviderContext.tsx:169]: I was looking at the code and I don't see it used anywhere. Probably an old artifact from the old desktop app.

File: apps/deploy-web/src/context/WalletProvider/WalletProvider.tsx [hunk 2]
```diff
@@ -108,6 +120,33 @@ export const WalletProvider = ({ children }) => {
...
+    }
+
+    selectWalletType(prev => (prev === "custodial" ? "managed" : "custodial"));
+  }
+
+  function connectManagedWallet() {
+    if (managedWallet) {
+      selectWalletType("managed");
+    } else {
+      create();
```
Comment (baktun14) [inline apps/deploy-web/src/context/WalletProvider/WalletProvider.tsx]: We could just rename the function to be self explanatory like `createManagedWallet`

**Review by baktun14 — APPROVED** [review]

File: apps/deploy-web/src/context/CertificateProvider/CertificateProviderContext.tsx (hunk 1 above)
Comment (ygrishajev) [reply apps/deploy-web/src/context/CertificateProvider/CertificateProviderContext.tsx:169]: Yea, it's possible. Didn't risk to remove. Should I?

Comment (baktun14) [reply apps/deploy-web/src/context/CertificateProvider/CertificateProviderContext.tsx:169]: I think it's safe to remove it. Maybe in a separate pr just to be safe.

**Pull request merged**